consensus/istanbul/ibft/core/core.go

```diff
@@ -98,9 +98,9 @@ func (c *core) finalizeMessage(msg *ibfttypes.Message) ([]byte, error) {
 	msg.Address = c.Address()
 
 	// Add proof of consensus
-	msg.CommittedSeal = []byte{}
 	// Assign the CommittedSeal if it's a COMMIT message and proposal is not nil
 	if msg.Code == ibfttypes.MsgCommit && c.current.Proposal() != nil {
+		msg.CommittedSeal = []byte{}
```

Comment:
is there a unit test we can add to reproduce this and avoid add a regression in the future?

Reply:
Done.